napari/utils/config.py

```diff
@@ -0,0 +1,51 @@
+"""Configuration.
```

Comment:
I like how simple this is. I think fine to use here for now, and when we make decisions on #1183 we can decide how to harmonize

Reply:
The reason I created `config.py` was being we were checking `NAPARI_ASYNC` in many places and I wanted to add `NAPARI_OCTREE` as a 2nd var. But I want to have that one imply async. I did not want people to have to set both.

If we did that with raw env vars everywhere it would be a mess. So `config.py` does it in one centralized place. Minor in a way, but it makes the various places keying off the config cleaner.

But if we move this to some future system as well. Anything that allows flags like these.